mlflow/utils/cli_args.py

```diff
@@ -80,17 +80,18 @@ def _resolve_env_manager(_, __, env_manager):
     return None
 
 
-def _create_env_manager_option(help_string):
+def _create_env_manager_option(help_string, default=None):
     return click.option(
         "--env-manager",
-        default=None,
+        default=default,
         type=click.UNPROCESSED,
         callback=_resolve_env_manager,
         help=help_string,
     )
 
 
 ENV_MANAGER = _create_env_manager_option(
+    default=_EnvManager.VIRTUALENV,
```

Comment:
Every cli command was using this default (by doing `env_manager = env_manager or _EnvMangager.VIRTUALENV`)